build/modules/aks/aks.tf

```diff
@@ -0,0 +1,115 @@
+# Copyright 2019 Google LLC All Rights Reserved.
```

Comment:
Have you thought about breaking this into multiple files to separate inputs / resources / outputs as per https://www.terraform-best-practices.com/code-structure#getting-started-with-structuring-of-terraform-configurations?